Hotfix for sync error due to missing concepts (#1324)
exercism/rust: concepts/hashmap/introduction.md

# Introduction

> Hashmap is a collection used to store key-value pairs. It allows arbitrary and user-defined types to be used as keys to index into it (technically any type that implement `Eq` and `Hash` traits can be used as keys). Hashmap have fast insertion and retrieval functions and are very useful in e.g., making frequency counts, storing relationships etc.,. Hashmap also features the powerful `entry` API that makes it ergonomic to use.

Reviewer: ```suggestion
Hashmap is a collection used to store key-value pairs. It allows arbitrary types to be used as keys and values. Hashmap have fast insertion and retrieval functions and are very useful in e.g., making frequency counts, storing relationships etc.,. Hashmap also features the powerful `entry` API that makes it ergonomic to use.
```

- arbitrary is a superset of user-defined
- arbitrary keys _and values_
- `Eq` and `Hash` bounds should be mentioned in `about`, not `introduction`
